org.scala-ide.sdt.core/src/org/scalaide/extensions/autoedits/ConvertToUnicode.scala

```diff
@@ -0,0 +1,36 @@
+package org.scalaide.extensions
+package autoedits
+
+import org.scalaide.core.text.Add
+import org.scalaide.core.text.Replace
+
+object ConvertToUnicodeSetting extends AutoEditSetting(
+  id = ExtensionSetting.fullyQualifiedName[ConvertToUnicode],
+  name = "Convert methods to their unicode representation",
+  description =
+    "Converts methods to their unicode representation if one exists." +
+    " This applies to =>, -> and <- which are converted to ⇒, → and ← respectively."
+)
+
+trait ConvertToUnicode extends AutoEdit {
+
+  override def setting = ConvertToUnicodeSetting
+
+  override def perform() = {
+    check(textChange) {
+      case Add(start, text) if text.size == 1 && start > 0 =>
```

Comment:
This probably won't work if the user selects a character and types a new one, effectively causing a `Replace` instead of an `Add`. Not a big issue, but it makes me wonder if the split between Replace/Add/Remove is actually useful.